pyprep/utils.py

```diff
@@ -108,6 +109,103 @@ def _mat_iqr(arr, axis=None):
     return iqr(arr, rng=np.clip(iqr_adj, 0, 100), axis=axis)
 
 
+def _eeglab_create_highpass(cutoff, srate):
+    """Create a high-pass FIR filter using Hamming windows.
+
+    Parameters
+    ----------
+    cutoff : float
+        The lower pass-band edge of the filter, in Hz.
+    srate : float
+        The sampling rate of the EEG signal, in Hz.
+
+    Returns
+    -------
+    filter : np.ndarray
+        A 1-dimensional array of FIR filter coefficients.
+
+    Notes
+    -----
+    In MATLAB PREP, the internal ``removeTrend`` function uses EEGLAB's
+    ``pop_eegfiltnew`` to high-pass the EEG data to remove slow drifts.
+    Because MNE's ``mne.filter.filter_data`` and EEGLAB's ``pop_eegfiltnew``
+    calculate filter parameters slightly differently, this function is
+    used to precisely match EEGLAB & MATLAB PREP's filtering method.
+
+    """
+    TRANSITION_WIDTH_RATIO = 0.25
+    trans_bandwidth = cutoff if cutoff < 2 else cutoff * TRANSITION_WIDTH_RATIO
+
+    # Calculate parameters for constructing filter
+    order = 3.3 / (trans_bandwidth / srate)
```

Comment:
where does the 3.3 come from? for magic numbers like these it's sometimes good style to do:

```
default_head_circumference_in_mm = 89
my_thing = default_head_circumference_in_mm * funky_tuple
```

instead of


```
my_thing = 89 * funky_tuple
```

Reply:
I honestly have no clue what the 3.3 is supposed to represent, it's not defined clearly in the EEGLAB code and my Google results haven't turned up a proper name for what it's supposed to be doing. The best I've found is Table 4.3 (page 7) in [this PDF](https://www.staff.ncl.ac.uk/oliver.hinton/eee305/Chapter4.pdf), which gives 3.3 as the unnamed constant to use when calculating the transition bandwidth for a Hamming window (here we're reversing that equation to get the filter order based on transition bandwidth & sample rate).

Does `HAMMING_CONSTANT` (or something to that effect) work for you?

Reply:
Ok, interesting. Yes that'd work - preferably with a comment that it's unclear